[Cache and Workers] Added a clarification on how to purge single URL with a custom cache key (#5413)
cloudflare/cloudflare-docs: content/cache/how-to/purge-cache.md

## Purge cache key resources

Purge resources that use Cache Keys via the [Cloudflare API](https://api.cloudflare.com/#zone-purge-files-by-url). If you use [Cloudflare’s Purge by URL](https://api.cloudflare.com/#zone-purge-files-by-url), include the headers and query strings that are in your custom Cache Key. 

> Currently, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker. Instead, use a [custom key created by Page Rules](/cache/how-to/create-cache-keys/). Alternatively, purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).

Reviewer: They can purge by tag, host or prefix.  It's only purge by URL that doesn't work.

> Currently, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker. Instead, use a [custom key created by Page Rules](/cache/how-to/create-cache-keys/). Alternatively, purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).

Reviewer 2: There are more options for purging with cache key than just cache tags. They can use flex purge options for a custom cache key. This would include: purge everything, purge by tag, purge by host and purge by prefix.

> At the moment, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker. Instead, use a [custom key created by Page Rules](/cache/how-to/create-cache-keys/). Alternatively, purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).

Reviewer 3: ```suggestion
Currently, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker. Instead, use a [custom key created by Page Rules](/cache/how-to/create-cache-keys/). Alternatively, purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).
```

> At the moment, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker, instead you should use a [custom key created by Page Rule](/cache/how-to/create-cache-keys/). Alternatively, you can also purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).

Reviewer 4: ```suggestion
At the moment, it is not possible to purge a URL stored through Cache API that uses a custom cache key set by a Worker. Instead, you should use a [custom key created by Page Rule](/cache/how-to/create-cache-keys/). Alternatively, you can also purge your assets using [cache-tags](/cache/how-to/purge-cache/#purge-using-cache-tags).
```

Reviewer 4: The sentence is too long. Hence the full stop to break it into more manageable bits. Wdyt?

Reviewer 3: Consider changing at the moment > currently for length
